tests/unit_test/conftest.py

```
@@ -0,0 +1,26 @@
+from unittest.mock import MagicMock, patch
```

Ruff (0.11.9)

`unittest.mock.MagicMock` imported but unused

Remove unused import

(F401)